README.md

```diff
@@ -865,6 +866,17 @@ $ node -e "import('./index.js').then(m => console.log(m.main))"
 [Function]
 ```
 
+### Enable source maps
+
+The `spago build` step generates source maps automatically.
+When bundling, you can include `--source-maps` to generate a final source map for your bundle.
+
+Example:
+```
+spago bundle -p my-project --source-maps --minify --outfile=bundle.js
+```
+will generate a minified bundle: `bundle.js`, and a source map: `bundle.js.map`.
+
```

Comment:
Ok, I'm not sure about node. 
But for chrome, it's automatic as long as your server serves the source map from the same directory as the js bundle. When you open the dev tools, chrome will make an additional network requests for source maps of your loaded assets. 
How do you want to summarize that?

Reply:
Ok it looks like node has support for source maps since version 12.12.0. It does require an additional flag, so I'll document it as well as the dev tools experience.

https://nodejs.org/en/blog/release/v12.12.0

Reply:
I think it would be fine to just link [this](https://developer.chrome.com/docs/devtools/developer-resources/) for Chrome, and just say that Chrome should pick it up if it can be found alongside your code